packages/cli/test/snapshot-matcher.js

```diff
@@ -51,17 +53,35 @@ function initializeSnapshots(snapshotDir) {
   });
 
   if (!process.env.UPDATE_SNAPSHOTS) {
+    process.on('exit', function printSnapshotHelp() {
+      if (snapshotErrors) {
+        console.log(
+          chalk.red(`
+Some of the snapshot-based test failed. Please carefully inspect
+the differences, your change may have introduced regressions.
+If the changes are intentional, then update snapshot files by
+running the tests with UPDATE_SNAPSHOTS=1 in the environment.
```

Comment:
I suggest that we rephrase the message as follows:
```
Some of the snapshot-based tests have failed. Please carefully inspect
the differences to prevent possible regressions.nIf the changes are
intentional, run the tests again with `UPDATE_SNAPSHOTS=1` environment
variable to update snapshots.
```